src/lib/abacus/index.ts

```diff
@@ -219,6 +228,17 @@ class AbacusStateManager {
     this.stateManager.transferStatus(hash, fromChainId, toChainId);
   };
 
+  // ------ Transactions ------ //
+
+  placeOrder = (callback: (p0: boolean, p1: Nullable<ParsingError>) => void) =>
+    this.stateManager.commitPlaceOrder(callback);
+
+  closePosition = (callback: (p0: boolean, p1: Nullable<ParsingError>) => void) =>
+    this.stateManager.commitClosePosition(callback);
+
+  cancelOrder = (orderId: string, callback: (p0: boolean, p1: Nullable<ParsingError>) => void) =>
+    this.stateManager.cancelOrder(orderId, callback);
+
   // ------ Utils ------ //
   placeOrderPayload = (): Nullable<HumanReadablePlaceOrderPayload> =>
```

Comment:
Are we able to deprecate these payload utils now?

Reply:
`placeOrderPayload` and `closePositionPayload` are still used

Reply:
removed the `cancelOrderPayload` method